gems/canvas_unzip/lib/canvas_unzip.rb

```diff
@@ -200,7 +200,7 @@ def size
     end
 
     # yields byte count
-    def extract(dest_path, overwrite=false, maximum_size=DEFAULT_BYTE_LIMIT)
+    def extract(dest_path, overwrite=false, maximum_size=DEFAULT_BYTE_LIMIT, digest_class: Digest::SHA256)
```

Comment:
Should the digest class depend on the setting?

Reply:
This a a gem, so I went with a generic default that was FIPS-compliant.  Canvas sets it when it's called.

Reply:
ok, makes sense